From dd69d0cec2e37964409b7ad5bd976a25f28c3532 Mon Sep 17 00:00:00 2001
From: Karl Berry <karl@freefriends.org>
Date: Sun, 5 Jan 2020 18:21:01 -0800
Subject: [PATCH 21/25] tests: Correctly simulate no emacs in
 t/nobase-nodist.sh

* t/nobase-nodist.sh: export EMACS, not EMCAS, as "no".
---
 t/nobase-nodist.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/nobase-nodist.sh b/t/nobase-nodist.sh
index 898ab9aa1..8966f9bcd 100644
--- a/t/nobase-nodist.sh
+++ b/t/nobase-nodist.sh
@@ -19,7 +19,7 @@
 . test-init.sh
 
 # Simulate no emacs and no python (it shouldn't be needed in this test).
-EMACS="no" PYTHON=":"; export EMCAS PYTHON
+EMACS="no" PYTHON=":"; export EMACS PYTHON
 
 cat >> configure.ac << 'END'
 AM_PATH_LISPDIR
-- 
2.25.0

